README.md

```
@@ -1,90 +1,192 @@
+- [Pre-requisite](#pre-requisite)
+  - [Software Dependencies](#software-dependencies)
+    - [npm (via asdf)](#npm-via-asdf)
+    - [yarn (via npm)](#yarn-via-npm)
+  - [Glific Backend](#glific-backend)
+- [Clone Frontend Repo](#clone-frontend-repo)
+  - [Git commands to clone](#git-commands-to-clone)
+- [Installation](#installation)
+  - [Start backend server](#start-backend-server)
+  - [Start frontend server](#start-frontend-server)
+  - [Login credentials](#login-credentials)
+  - [Configure Gupshup settings](#configure-gupshup-settings)
```

markdownlint-cli2 (0.17.2)

Link fragments should be valid
null

(MD051, link-fragments)